tests/bluetooth/host/buf/cmake.txt

```diff
@@ -15,6 +15,7 @@ FILE(GLOB module_mocks
   ${ZEPHYR_BASE}/tests/bluetooth/host/buf/mocks/net_buf.c
   ${ZEPHYR_BASE}/tests/bluetooth/host/buf/mocks/iso.c
   ${ZEPHYR_BASE}/tests/bluetooth/host/buf/mocks/hci_core.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/buf/mocks/buf_help_utils.c
```

Comment:
I would rename to `net_buf_expects` or `net_buf_validator`, something more descriptive than utils.

Reply:
I'd say to create a new `net_buf_expects` `.h` and `.c` pair to contain the current implementation and remove `buf_help_utils.c`
This because `buf_help_utils.h` contains some unrelated definitions to `net_buf`.

Reply:
yes, I hadn't realized the pool getters were in there too